mithril-aggregator/tests/certificate_chain.rs

```diff
@@ -165,38 +165,25 @@ async fn certificate_chain() {
     comment!(
         "Signers register & send signatures, the new certificate should be link to the first of the previous epoch"
     );
-    tester.register_signers(&new_signers).await.unwrap();
-    let signed_entity_type = observer
-        .get_current_signed_entity_type(SignedEntityTypeDiscriminants::MithrilStakeDistribution)
-        .await
-        .unwrap();
+    tester.register_signers(&next_signers).await.unwrap();
     tester
-        .send_single_signatures(&signed_entity_type, &signers)
+        .send_single_signatures(
+            SignedEntityTypeDiscriminants::MithrilStakeDistribution,
+            &signers,
+        )
         .await
         .unwrap();
     cycle!(tester, "ready");
-    let (last_certificates, snapshots) =
-        tester.get_last_certificates_and_snapshots().await.unwrap();
-    assert_eq!((5, 2), (last_certificates.len(), snapshots.len()));
-    assert_eq!(
-        (
-            last_certificates[0].beacon.immutable_file_number,
-            last_certificates[0].beacon.epoch,
-        ),
-        (
-            last_certificates[1].beacon.immutable_file_number,
-            last_certificates[1].beacon.epoch + 1,
-        ),
-        "Only the epoch should have changed"
-    );
-    assert_eq!(
-        &last_certificates[0].previous_hash, &last_certificates[4].hash,
-        "The new epoch certificate should be linked to the first certificate of the previous epoch"
-    );
-    assert_eq!(
-        last_certificates[0].metadata.get_stake_distribution(),
-        last_certificates[2].metadata.get_stake_distribution(),
-        "The stake distribution update should only be taken into account at the next epoch",
+    // Note: the new stake distribution will only be taken into account in the next epoch
```

Comment:
What is the purpose of this comment?

Reply:
It's a reminder as why it's still the first fixture that's used in the following `ExpectedCertificate` and not the `next_fixture`.
I put it because when writing the test I was still confused about the epoch offsets.